akka-http-core/src/test/scala/akka/http/impl/engine/parsing/RequestParserBodyPartSpec.scala

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.impl.engine.parsing
+
+import akka.actor.ActorSystem
+import akka.http.impl.engine.parsing.ParserOutput._
+import akka.http.impl.util._
+import akka.http.scaladsl.model.Multipart.General.Strict
+import akka.http.scaladsl.model.headers.{ ContentDispositionTypes, `Content-Disposition` }
+import akka.http.scaladsl.model.{ ParsingException ⇒ ModelParsingException, _ }
+import akka.http.scaladsl.settings.ParserSettings
+import akka.http.scaladsl.util.FastFuture
+import akka.stream.ActorMaterializer
+import akka.stream.TLSProtocol._
+import akka.stream.impl.fusing.IteratorInterpreter
+import akka.stream.scaladsl._
+import akka.testkit.TestKit
+import akka.util.ByteString
+import com.typesafe.config.{ Config, ConfigFactory }
+import org.scalatest.matchers.Matcher
+import org.scalatest.{ BeforeAndAfterAll, FreeSpec, Matchers }
+
+import scala.collection.immutable.{ VectorBuilder, Seq ⇒ ImmutableSeq }
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+case class RequestParserBodyPartSpecMode(name: String, lineFeed: String)
+
+// Define a trait to duplicate all tests for CRLF and LF
+trait RequestParserBodyPartSpec extends FreeSpec with Matchers with BeforeAndAfterAll {
```

Comment:
I think we should reuse the existing specs at `akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/MultipartUnmarshallersSpec.scala`. These once won't work anymore on Akka 2.5 because the `IterationInterpreter` is gone.